dds/DCPS/Serializer.h

```diff
@@ -298,6 +300,39 @@ size_t serialized_size(const Encoding& encoding, const T& value)
   return size;
 }
 
+/**
+* This helper class can be used to construct ace message blocks from OctetSeqs
+* and be used with the Serializer to serialize/deserialize directly into the OctetSeq buffer.
+* The OctetSeq must have its length set before constructing this object.
+*/
+class OpenDDS_Dcps_Export MessageBlockHelper {
+  public:
```

Comment:
All the contents gained an extra layer of indentation somehow, please remove it.